docs/forms.rst

```diff
@@ -75,6 +80,10 @@ Used on `account_signup <views.html#signup-account-signup>`__ view.
             # You must return the original result.
             return user
 
+The configuration setting ``ACCOUNT_SIGNUP_FORM_CLASS`` is the same as
+inheriting the SignupForm class. It is recommended to inherit and use the
+setting below instead.
```

Comment:
Why would overriding the `signup` form complete be preferable over using ACCOUNT_SIGNUP_FORM_CLASS ?

Reply:
The project currently has 2 separate ways of supplying the signup form class, `ACCOUNT_SIGNUP_FORM_CLASS` and `ACCOUNT_FORMS['signup']`.

Is the single dict `ACCOUNT_FORMS = {...}` not a cleaner way of supplying the forms that are being overridden throughout the application. Now all forms that are overridden are in a single location in the settings file rather than potentially scattered and using different slightly naming schemes.